parser/protobuf_usp.go

```diff
@@ -0,0 +1,152 @@
+package parser
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"sync"
+	"sync/atomic"
+
+	"github.com/gogo/protobuf/proto"
+	"github.com/telenornms/skogul"
+	"github.com/telenornms/skogul/gen/usp"
+)
+
+type P struct {
```

Comment:
Considering this is global to all parsers, I'd prefer a slightly more descriptive name, e.g.: USP or something like that.